apps/docs/pages/guides/functions/debugging.mdx

```diff
@@ -96,6 +155,15 @@ Examine if there are any unnecessary or redundant dependencies that can be remov
 deno info --import-map=/path/to/import_map.json /path/to/function/index.ts
 ```
 
+## Limitations
+
+- Outgoing connections to ports `25`, `465`, and `587` are not allowed.
+- Serving of HTML content is not supported (`GET` requests that return `text/html` will be rewritten to `text/plain`).
+- Deno and Node file system APIs are not available.
+- Memory limit for the function: 150 MB (heap) + 150 MB (external - array buffers/ WASM)
+- Wall clock limit: 400 s
+- CPU Time: 200 s
```

Comment:
@monicakh this has to be 2s

Reply:
Fix here: https://github.com/supabase/supabase/pull/20258